internals/secrethub/init.go

```diff
@@ -105,12 +121,98 @@ func (cmd *InitCommand) Run() error {
 	case InitModeSignup:
 		signupCommand := SignUpCommand{
 			io:              cmd.io,
-			newClient:       cmd.newClient,
+			newClient:       cmd.newUnauthenticatedClient,
 			credentialStore: cmd.credentialStore,
 			progressPrinter: cmd.progressPrinter,
 			force:           cmd.force,
 		}
 		return signupCommand.Run()
+	case InitModeSetupCode:
+		setupCode := cmd.setupCode
+		if setupCode == "" {
+			var err error
+			setupCode, err = ui.AskAndValidate(cmd.io, "What is your setup code?\n", 3, api.ValidateSetupCode)
+			if err != nil {
+				return err
+			}
+		}
+
+		fmt.Fprintf(
+			cmd.io.Output(),
+			"An account credential will be generated and stored at %s. "+
+				"Losing this credential means you lose the ability to decrypt your secrets. "+
+				"So keep it safe.\n",
+			credentialPath,
+		)
```

Comment:
I think this is also in the signup, right? Can we de-duplicate it?

Reply:
I made the message a constant, which is the most meaningful way I could deduplicate this.